doc/frameworks/pytorch/using_pytorch.rst

```diff
@@ -212,7 +212,8 @@ with the ``pytorchddp`` option as the distribution strategy.
 .. note::
 
   This PyTorch DDP support is available
-  in the SageMaker PyTorch Deep Learning Containers v1.12 and later.
+  in the SageMaker PyTorch Deep Learning Containers v1.11 and later
+  on GPU instances.
```

Comment:
```suggestion
  for training on GPUs.
```

Reply:
are we dropping CPU support? is it because the underlying launcher will error out for CPU instance? Let's make clear here what types of instances are supported. 
also curious on what happens for gloo backend on CPU instances?